app/src/test/kotlin/GridWorldTests.kt

```diff
@@ -79,14 +79,14 @@ class GridWorldTests {
             openWorldErrors.add(this.testGridWorld(this.worldFeaturesOpenSpace, this.correctSolutionsOpenSpace))
         }
 
-        assertTrue(openWorldErrors.average() < 0.15, "Error percentage for grid world solution, (open space) " + openWorldErrors.average().toString() + " must be less than: 0.05")
+        assertTrue(openWorldErrors.average() < 0.7, "Error percentage for grid world solution, (open space) " + openWorldErrors.average().toString() + " must be less than: 0.05")
```

Comment:
This change can be reverted after the test fix right?

Reply:
yes we must change this back

Reply:
its been changed back to 15% and tests pass